pkg/asset/installconfig/platformprovisioncheck.go

```diff
@@ -11,8 +11,8 @@ import (
 	gcpconfig "github.com/openshift/installer/pkg/asset/installconfig/gcp"
 	kubevirtconfig "github.com/openshift/installer/pkg/asset/installconfig/kubevirt"
 	osconfig "github.com/openshift/installer/pkg/asset/installconfig/openstack"
-	ovirtconfig "github.com/openshift/installer/pkg/asset/installconfig/ovirt"
```

Comment:
This import is removed here and added back in a subsequent commit; those changes should be squashed into a single commit and there should not be a case where a commit is knowingly broke. If we merged these commits then revert the second one, we would be in a broken state.